### ISSUE TITLE
Transaction not confirming

### ISSUE THREAD

**Issue by battsogtRising** [iotaledger/wallet#502]
I've transferred some IOTAs from my Windows wallet 2.5.3 to Binance. 
It has been almost 3 days. But transaction is still pending.
I've tried reattaching multiple times. It does nothing but increasing the transaction history log.

**Comment by wonkytonky** [comment]
update wallet 2.5.4 first try again reattaching

**Comment by pascalandy** [comment]
same issue since 3 days.

**Comment by bartolosimpson** [comment]
Yes, update wallet to lattest version, reattach with ccurl version.

**Comment by pascalandy** [comment]
Ohh, using **ccurl** version it's now attaching !

**Comment by thivaz** [comment]
how to reattach with ccurl ver?
